saw-remote-api/python/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Revision history for saw-client
 
-## 0.9.1 -- YYYY-MM-DD
+## 0.9.2 -- 2022-05-09
+
+* Added two new commands, `eval_int` and `eval_bool`, that evaluate Cryptol
+  expressions to Python integers or booleans.
+
+## 0.9.1 -- 2021-12-21
```

Comment:
I don't think `0.9.1` has been released yet, has it? If not, best to just add these release notes under `0.9.1`.

Reply:
I don't think we've ever formally written up the conventions anywhere, but my understanding is that we use odd version numbers for development versions and even version numbers for release versions, similar to GHC's versioning conventions.